### PR TITLE
chore: Typescript Cleanup Part 3 of 3 for New error errors after Version change

### PR DESCRIPTION
**Pull request by v-prachin**
#### Details

Typescript Cleanup Part 3 of 3 
We have total 90 errors remaining in 53 files.

##### Motivation
Resolved errors in all these files:

1. src/tests/unit/tests/DetailsView/components/adhoc-static-test-view.test.tsx (1)
2. src/tests/unit/tests/DetailsView/components/adhoc-tab-stops-test-view.test.tsx (1)
3. src/tests/unit/tests/DetailsView/components/iframe-warning.test.tsx (2)
4. src/tests/unit/tests/DetailsView/components/issues-table.test.tsx (1)
5. src/tests/unit/tests/DetailsView/details-view-container.test.tsx (2)
6. src/tests/unit/tests/injected/adapters/resolution-creator.test.ts (1)
7. src/tests/unit/tests/reports/package/axe-results-report.test.ts	(1)
8. src/tests/unit/tests/background/target-tab-controller.test.ts (1)
9. src/tests/unit/tests/background/stores/persistent-store.test.ts	(1)
10. src/tests/unit/tests/DetailsView/components/adhoc-issues-test-view.test.tsx(1)
11. src/tests/unit/tests/background/initial-visualization-store-data-generator.test.ts (2)
12. src/tests/unit/tests/reports/components/inline-image.test.tsx (1)

Number of errors covered by me as part of this user story: **15**

**Incorporated changes from Saanica's PR related to Feature 3**

1. src/tests/unit/tests/common/message-creators/assessment-card-selection-message-creator.test.ts
2. src/tests/unit/tests/common/message-creators/card-selection-message-creator.test.ts
3. src/tests/unit/tests/common/message-creators/needs-review-card-selection-message-creator.test.ts
4. src/tests/unit/tests/common/notification-creator.test.ts
5. src/tests/unit/tests/common/telemetry-data-factory.test.ts
6. src/tests/unit/tests/DetailsView/actions/details-view-action-message-creator.test.ts
7. src/tests/unit/tests/DetailsView/actions/tab-stop-requirement-action-message-creator.test.ts

Number of errors covered by Saanica: **14**

Thus, total number of errors covered as part of this user story: **29**

#### Pull request checklist

- [x] Addresses an existing issue: #(https://github.com/microsoft/accessibility-insights-web/pull/6611)
- [x] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: src/tests/unit/tests/injected/adapters/resolution-creator.test.ts [hunk 1]
```diff
@@ -62,8 +62,6 @@ describe('ResolutionCreator', () => {
                 ],
                 all: [{ id: null, message: 'all 1 message', data: null }],
                 none: [{ id: null, message: 'none 1 message', data: null }],
-                html: 'test html',
```
Comment (v-viyada) [inline src/tests/unit/tests/injected/adapters/resolution-creator.test.ts:65]: why this is removed?

File: yarn.lock [hunk 2]
```diff
@@ -3339,7 +3339,7 @@ __metadata:
     eslint-plugin-import: ^2.29.1
     eslint-plugin-react: ^7.33.2
     eslint-plugin-security: ^1.7.1
-    express: ^4.18.2
```
Comment (v-viyada) [inline yarn.lock:3342]: why there are yarn lock changes.

**Review by v-viyada — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (v-prachin) [reply yarn.lock:3342]: In accessibility insights Main repo this version is upgraded to 4.19.2. The branch typescript-part5 was not upto date with main thus this change is showing.

File: src/tests/unit/tests/injected/adapters/resolution-creator.test.ts (hunk 1 above)
Comment (v-prachin) [reply src/tests/unit/tests/injected/adapters/resolution-creator.test.ts:65]: NodeResolutionData Inteface does not consist of html property, checked git history. Thus, removed the property.

File: yarn.lock (hunk 2 above)
Comment (v-prachin) [reply yarn.lock:3342]: For package.json and yarn lock taken latest changes from typescript-part5 branch and pasted as it is in these files. So these changes are not due to fixing errors

**Pull request closed without merging**